.travis.yml

```diff
@@ -24,9 +24,9 @@ before_install:
   - pip install mako servo-tidy
 script:
   - servo-tidy
-  - if [ $BUILD_KIND = DEBUG ]; then (cd webrender_traits && cargo test --verbose --features "ipc"); fi
-  - if [ $BUILD_KIND = DEBUG ]; then (cd webrender_traits && cargo test --verbose); fi
+  - if [ $BUILD_KIND = DEBUG ]; then (cd webrender_traits && RUST_BACKTRACE=1 cargo test --verbose --features "ipc"); fi
```

Comment:
let's remove `RUST_BACKTRACE=1` from here, given that `env` appears to be working?